components/layout/Login/LoginContent/LoginContent.spec.tsx

```diff
@@ -7,8 +7,6 @@ import userEvent from '@testing-library/user-event'
 
 import * as stories from './LoginContent.stories' // import all stories from the stories file
 
-import type { UserEvent } from '@testing-library/user-event/dist/types/setup/'
```

Comment:
don't modify these core components test files inside the fork, add in the core repo

Reply:
Resolved